### PR TITLE
Updated naming conventions around domain names, part 6

### PR DESCRIPTION
**Pull request by orangejenny**
@millerdev

### CODE REVIEW

File: corehq/apps/users/views/__init__.py [hunk 1]
```diff
@@ -940,18 +940,18 @@ def page_url(self):
...
                 initial.update({
                     'email': self.request.user.get_username(),
                     'full_name': self.request.user.get_full_name(),
                 })
             self.request_form = DomainRequestForm(initial=initial)
         return {
-            'domain': domain.name,
-            'domain_name': domain.display_name(),
+            'domain': domain_obj.name,
+            'domain_name': domain_obj.display_name(),
```
Comment (millerdev) [inline corehq/apps/users/views/__init__.py]: This is lovely (although not made worse by this PR)... adding to the confusion about what `domain_name` means (I thought it normally was the same as `domain.name`).

File: corehq/apps/users/views/__init__.py [hunk 2]
```diff
@@ -967,9 +967,9 @@ def post(self, request, *args, **kwargs):
                     domain_request = DomainRequest(**data)
                     domain_request.send_request_email()
                     domain_request.save()
-                    domain = Domain.get_by_name(domain_request.domain)
+                    domain_obj = Domain.get_by_name(domain_request.domain)
                     return render(request, "users/confirmation_sent.html", {
-                        'hr_name': domain.display_name() if domain else domain_request.domain,
+                        'hr_name': domain_obj.display_name() if domain_obj else domain_request.domain_obj,
```
Comment (millerdev) [inline corehq/apps/users/views/__init__.py]: The last one on this line should not be changed. `DomainRequest.domain` is a string, and `DomainRequest.domain_obj` is not a thing.

https://github.com/dimagi/commcare-hq/blob/d7301f0bac0bdd33124f294d9067c86e9617bbd5/corehq/apps/users/models.py#L2512

**Review by millerdev — COMMENTED** [review]
This is great! Thank you for taking the initiative to do it.

File: corehq/apps/users/views/__init__.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/users/views/__init__.py]: There's something of a convention of using `hr_name` (human-readable name). Updated in e109388cc4b

File: corehq/apps/users/views/__init__.py (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/users/views/__init__.py]: thanks - 3e83f6454f2

**Review by millerdev — APPROVED** [review]

**Pull request merged**